### PR TITLE
README: add info and fix typos

### PR DESCRIPTION
**Pull request by CaitinChen**
### What problem does this PR solve? 

add info and fix typos

### What is changed and how it works?

add info and fix typos
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

**Comment by CaitinChen** [comment]
@lilin90 PTAL

**Review by nrc — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -6,7 +6,21 @@
...
+`tiup` is a tool to download and install [TiDB](https://docs.pingcap.com/tidb/stable/overview) components.
+
+## Documentation
+
+- [English](https://docs.pingcap.com/tidb/stable/tiup-documentation-guide)
+- [简体中文](https://docs.pingcap.com/zh/tidb/stable/tiup-documentation-guide)
+
+## Blog
+
+- [English](https://www.pingcap.com/blog/)
```
Comment (lilin90) [inline README.md:18]:
```suggestion
- [English](https://pingcap.com/blog/)
```

Comment (CaitinChen) [inline README.md:18]:
```suggestion
- [English](https://pingcap.com/blog/)
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**